mod/edit/set/all/form_buttons.rb

```diff
@@ -3,6 +3,19 @@ def standard_submit_button
     output [standard_save_button, standard_save_and_close_button]
   end
 
+# Generates a standard save button with optional parameters.
+#
+# @param [Hash] opts The options for the save button.
+# @option opts [String] :text ("Save") The text displayed on the button.
+# @option opts [String] :data-cy ("save") The value for the 'data-cy' attribute
+#   used for testing or identification purposes.
```

Comment:
this option is not correct.  It's pretty much the only thing that _won't_ get passed on to `submit_button`